ball.ai/src/main/java/com/google/sps/servlets/PlayerInfoServlet.java

```diff
@@ -0,0 +1,37 @@
+//package com.google.sps.servlets;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.FileInputStream;
+import java.util.List;
+import java.net.URL;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/playerInfo")
+public class PlayerInfoServlet extends HttpServlet {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html");
+        String fileName = "players.csv";
+        File file = new File(url.getPath());
+        try{
+            Scanner inputStream = new Scanner(file);
+            while(inputStream.hasNext()){
```

Comment:
Imagine this is reading the CSV, but would be cool if you could comment and make it more explicit